### ISSUE TITLE
Replicate Layout Error List index out of range

### ISSUE THREAD

**Issue by BillyPocketNC** [MitjaNemec/Kicad_action_plugins#126]
![image](https://user-images.githubusercontent.com/48296685/112401977-45adbe80-8cd1-11eb-8141-a2683b716f85.png)
This message seems a little vague.

This error popped up when I tried to replicate a circuit where one of the chips in the master/source layout had more pins than one of the other layouts. (I modified a heatsink pad to have the required Vias as pictured below)
![image](https://user-images.githubusercontent.com/48296685/112402381-1055a080-8cd2-11eb-95e0-81ec0ceedb1c.png)
Updating all the footprints resolved the issue.

replicate_layout log:
[replicate_layout.log](https://github.com/MitjaNemec/Kicad_action_plugins/files/6201240/replicate_layout.log)

**Comment by MitjaNemec** [comment]
Yeah, the error message should be better, but besides figuring out that some footprints don't match between source and destination layout I don't think I don't think I can do.

Thanks for the report and the log. It is much appreciated

**Comment by BillyPocketNC** [comment]
No problem. Thanks for the work you put into these plugins. They have saved me hours.
